### PR TITLE
Nadade/docker machine setup

### PR DESCRIPTION
**Pull request by nadade**

### CODE REVIEW

File: install/docker-install-dockertoolbox.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    fi
+
+    if [ ! -f /usr/local/etc/bash_completion ]; then
+        echo "Installing bash completion..."
+        brew install bash-completion
+        if [ ! -f ~/.bash_profile ]; then
+            > ~/.bash_profile
+        fi
+        echo "Adding bash completion to .bash_profile"
+        { echo "if [ -f $(brew --prefix)/etc/bash_completion ]; then"; echo "  . $(brew --prefix)/etc/bash_completion"; echo "fi"; } >> ~/.bash_profile
```
Comment (kuba-kubula) [inline install/docker-install-dockertoolbox.sh]: @nadade I'm not sure about the dot character just in front of the second `brew --prefix` piece of this line above

Comment (kylef) [inline install/docker-install-dockertoolbox.sh]: . is shorthand for `source` in bash and zsh.

The following is equivalent:

```shell
. $(brew --prefix)/etc/bash_completion
source $(brew --prefix)/etc/bash_completion
```

File: install/docker-install-dockertoolbox.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    which brew > /dev/null 2>&1
+    if [ $? -eq 1 ]; then
+        echo "Homebrew not installed on your machine. Press return to install it"
+        read whatever
+        ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+    else
+        brew update
+    fi
+
+    if [ ! -f /usr/local/etc/bash_completion ]; then
```
Comment (kylef) [inline install/docker-install-dockertoolbox.sh]: Would love if this first checked you are using Bash, people who do not use bash (myself) probably don't want bash stuff installed on their system.

```
$ echo $SHELL
/usr/local/bin/fish
```

Comment (kylef) [inline install/docker-install-dockertoolbox.sh]: -> #14

File: install/docker-install-dockertoolbox.sh (hunk 1 above)
Comment (kuba-kubula) [inline install/docker-install-dockertoolbox.sh]: @kylef `source` is a lot more transparent for me and also maybe others. But... YMMV.

File: install/docker-install-dockertoolbox.sh (hunk 2 above)
Comment (nadade) [inline install/docker-install-dockertoolbox.sh]: thanks, @kylef!

File: install/docker-install-dockertoolbox.sh (hunk 1 above)
Comment (nadade) [inline install/docker-install-dockertoolbox.sh]: @kylef: agreed, I'll change it

**Pull request merged**